common/changes/@uifabric/example-app-base/jg-example-theming_2018-10-11-23-05.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/example-app-base",
+      "comment": "Add optional app customizations and apply to examples when provided.",
+      "type": "minor"
+    }
+  ],
+  "packageName": "@uifabric/example-app-base",
+  "email": "jason.gore@grabtaxi.com"
+}
```

Comment:
?!?!

Reply:
Ugh that's what I get when I push from home

Reply:
haha